src/op_mode/vrrp.py

```diff
@@ -35,7 +36,16 @@
 
 args = parser.parse_args()
 
+def is_configured():
+    """ Check if VRRP is configured """
+    config = ConfigTreeQuery()
+    if not config.exists(['high-availability', 'vrrp', 'group']):
+        return False
+
```

Comment:
Please add return True for the other control path

Reply:
Done